cylc/flow/graph_parser.py

```diff
@@ -273,8 +290,29 @@ def parse_graph(self, graph_string):
         # Parameterization can duplicate some dependencies, so use a set.
         pairs = set()
         for line in line_set:
+            chain = []
             # "foo => bar => baz" becomes [foo, bar, baz]
-            chain = line.split(ARROW)
+            # "foo_-32768 => bar" becomes [bar] (remove out-of-range nodes)
```

Comment:
Or would this line be completely removed?

Reply:
If the dep is out of bounds then `[bar]` sounds right to me.